webpack.config.js

```diff
@@ -308,6 +333,17 @@ if (TARGET !== 'readme') {
             minRatio: 0.8
         }),
 
+        // Statics from build.
+        // The first one should come from user runtime environment instead.
```

Comment:
What do you mean by "come from the user runtime environment"? Is that a build we expect the user to run? or is it the webbrowser environment? (do I have any other choices?)

Reply:
Fallback should be `en`. Imposing `en` as default forces unlogged screens to follow that, even if a local admin has set his configuration to use something else.

Reply:
Sorry if I'm dense, but I still partially follow: as far as I understand the situation, you want the default to be `en`, except when the local admin has set a different language. I suppose that setting will be through `ledgersmb.conf`? Then your remark is referring to the language set in 'ledgersmb.conf' which needs to provide the default. And that `en` should be the fallback if no default was provided in `ledgersmb.conf`?

Maybe we can simplify things, because the default when no language is set in `ledgersmb.conf` already *is* `en`: see https://github.com/ledgersmb/LedgerSMB/blob/master/lib/LedgerSMB/Sysconfig.pm#L293

Reply:
Webpack has no access to the Perl code, nor the user ledgersmb.conf during compile, so I need to provide a sane default and then load the proper language at runtime.
If no language was set as default, then everything would be handled by the fallback entry

Reply:
Ah! It's mostly a dummy setting! Now I get it. But should it say "we're providing a default here, but the true default will come at runtime from the browser context"?

Reply:
Not yet from the browser context. Default and fallback are set to `en`. When logging, the user preferences will provide the input to pull the proper locale.